### PR TITLE
Simplify Placement Policy

### PR DESCRIPTION
**Pull request by realloc**
Proposed protobuf format for Placement Policy simplifies direct editing in visual editors and import/export from other formats like JSON.

### CODE REVIEW

File: netmap/types.proto [hunk 1]
```diff
@@ -5,89 +5,96 @@ package neo.fs.v2.netmap;
...
+  string key = 2;
+
+  // Filtering operation
+  Operation op = 3;
+
+  // Value to match
+  string value = 4;
+
+  // Next Filter logical relation
+  Operation next_op = 5;
```
Comment (fyrchik) [inline netmap/types.proto:54]: Shouldn't this be also a `Filter`? What value does this operation applies to?

Comment (realloc) [reply netmap/types.proto:54]: That's exactly not to use Filter here and have chain of Filters without recursion.

Comment (fyrchik) [reply netmap/types.proto:54]: Does the semantics of `(a AND) (b OR) (c AND) (d *)` correspond to the DNF `(a AND b) OR (c AND d)` or to the `((a AND b) OR c) AND d`?

**Comment by realloc** [comment]
Latest PR update proposes to have nested filters, but only top level filter can be named.

Example in JSON:
```json
{
  "name": "BigDataInJapan",
  "op": "AND",
  "filters": [
    {
      "op": "OR",
      "filters": [
        {
          "key": "PowerSupply",
          "op": "GT",
          "value": "9000"
        },
        {
          "key": "BigDataStorage",
          "op": "EQ",
          "value": "true"
        },
      ]
    },
    {
      "key": "Country",
      "op": "EQ",
      "value": "Japan"
    }
  ]
}
```

Same higher level Operation applies to all lower level filters.

Example in JSON:
```json
  {
    "name": "CatStor",
    "op": "OR",
    "filters": [
      {
        "key": "StorageType",
        "op": "EQ",
        "value": "SSD"
      },
      {
        "name": "GoodOldEurope"
      },
      {
        "name": "RAMDisk"
      }      
    ]
  }
```

**Review by alexvanin — APPROVED** [review]
Seems good and clean.

**Pull request merged**